### ISSUE TITLE
上传音频报错

### ISSUE THREAD

**Issue by Song367** [tiankuan93/ComfyUI-V-Express#15]
Failed to validate prompt for output 20:
* Load_Audio_Path 6:
  - Value not in list: audio_path: '春福绿茶，唤醒 (1).MP3' not in []
Output will be ignored
Failed to validate prompt for output 10:
Output will be ignored
invalid prompt: {'type': 'prompt_outputs_failed_validation', 'message': 'Prompt outputs failed validation', 'details': '', 'extra_info': {}}

**Comment by adambnn** [comment]
yes, the load audio path node is broken, I am using load audio path from video instead.

**Comment by adambnn** [comment]
if you go to web\js\VEuploadAudio.js you will find the 3 imports are missing. There are no js in the scripts folder, only a extract_kps_sequence_and_audio.py

import { app } from "../../../scripts/app.js";
import { api } from "../../../scripts/api.js";
import { ComfyWidgets } from "../../../scripts/widgets.js";
